index.md

```diff
@@ -40,6 +40,7 @@ out of the process. Here are a few of our favorites:
 * [Mathias Bynens' dotfiles](https://github.com/mathiasbynens/dotfiles) includes a bootstrap script that rsyncs your repo to your home folder. Mathias' [macOS defaults script](https://github.com/mathiasbynens/dotfiles/blob/master/.macos) is legendary.
 * [Matt Smith's dotfiles](https://github.com/mattorb/dotfiles) includes a one-liner install, fish, vscode, mac desktop app installs via brew cask, os x customizations inspired by Mathias, etc.  No dotfiles framework, just shell scripts to set everything up.
 * [mihaliak's dotfiles](https://github.com/mihaliak/dotfiles) for macOS focused on Web development
+* [Mohit Singh's dotfiles](https://github.com/mohitsinghs/dotfiles) has a lighweight setup for JavaScript devlopment environment on macOS.
```

Comment:
s/lighweight/lightweight/

Reply:
Not only that,
s/devlopment/development/
I'm ashamed.
Learned a lesson today. Can't thank you enough.

Made another pr #201